src/MentionsInputSelectors.spec.tsx

```
@@ -694,6 +702,49 @@ describe('MentionsInputSelectors', () => {
     })
   })
 
+  it('normalizes grouped provider sections into flat items and section metadata', () => {
+    const users = [
+      { id: 'alice', display: 'Alice' },
+      { id: 'adam', display: 'Adam' },
+    ]
+    const teams = [{ id: 'frontend', display: 'Frontend Team' }]
+    const publicSection: MentionDataSection = {
+      label: 'Public API section',
+      items: [],
```

ESLint

[error] Property `items` is defined but never used.

(unicorn/no-unused-properties)